### ISSUE TITLE
When installing Zap Desktop 0.2.1 Beta Error: LND has unexpectedly quit

### ISSUE THREAD

**Issue by d4amenace** [LN-Zap/zap-desktop#674]
![example2](https://user-images.githubusercontent.com/29969146/44168411-cd9da780-a096-11e8-8d5b-42c4d373d9cb.png)

Same point every time...... Happens just after I unlock wallet.  I had installed an instance of lnd a while back but it is not running..(at least according to task manager).  It shuts itself down before getting to syncing point.

**Comment by d4amenace** [comment]
I am running windows 10.

**Comment by mrfelton** [comment]
@d4amenace please could you try removing your old Zap userdata before starting Zap.

On windows this can be found at `C:\\Users\user\AppData\Roaming\ZapDesktop`

**Comment by evertvr** [comment]
Running Windows 7, having the same issue.

Removed the userdata, but still same error

**Comment by evertvr** [comment]
Had the same error, for me it was some other service was running on 127.0.0.1:8080

**Comment by skcheffy** [comment]
Just installed on Mac and it looks like i am getting the error.  See picture.

<img width="1106" alt="screen shot 2018-08-17 at 7 29 43 pm" src="https://user-images.githubusercontent.com/31999006/44292710-60377580-a254-11e8-8076-154c73347f90.png">

**Comment by mrfelton** [comment]
@skcheffy could you test with the PR at https://github.com/LN-Zap/zap-desktop/pull/683 in place please? This should enhance that error message with some more detail about why specifically lnd exited.

Additionally - do you have something else already running on port `8080` or `90001`?

**Comment by skcheffy** [comment]
Sorry not as tech savvy as I had hoped yet.

No I checked ports and nothing opened there.  How do I test with PR #683 in place?  I went there but not sure exactly what to do.  Do I need to put a file somewherE?

**Comment by skcheffy** [comment]
So I have uninstalled ZAP on my Mac and removed all files associated on my computer via OmniSweep.

Currently re-installed and waiting for sync but no error.  I assume there was something casusing the issue in a folder I found hidden.  

Keep you posted

**Comment by skcheffy** [comment]
It seems like doing the above was successful and worked.

**Comment by patmillertime** [comment]
@JimmyMow @mrfelton do you have any suggestions for resolving the problem I am experiencing above? Would you like me to open a new issue?

**Comment by mrfelton** [comment]
@patmillertime please could you first update to 0.3.2 (released just now) and confirm if the issue persists.

If so, you can try starting up the app with debugging enabled. From the terminal:

```
DEBUG=zap* DEBUG_LEVEL=debug /Applications/ZapDesktop.app/Contents/MacOS/ZapDesktop
```

This will print lots of debugging information on the terminal, in which you may be able to get some more context about why lnd is crashing. Please share your logs leading up to the point that the app crashes. Thanks.

**Comment by patmillertime** [comment]
Thanks @mrfelton. I will do this when I get home this evening and report back!

**Comment by qjawe** [comment]
zap:lnd [INF]  2018-12-29 08:45:31.472 [INF] CRTR: FilteredChainView starting +0ms
  zap:lnd [ERR]  2018-12-29 08:45:31.481 [ERR] DISC: Unable to rebroadcast stale channels: unable to retrieve outgoing channels: Channel from self node has no policy +1ms
  zap:lnd [DBG]  2018-12-29 08:45:31.487 [DBG] BTCN: Waiting for filter headers (height=1449938) to catch up the rescan start (height=1449938) +0ms
  zap:lnd [DBG]  2018-12-29 08:45:31.487 [DBG] BTCN: Starting rescan from known block 1449938 (00000000000000da53cb40da20776312c728970d5bb81ea7c942b1b679ea3e0f) +0ms
  zap:lnd [DBG]  2018-12-29 08:45:31.487 [DBG] BTCN: Rescan became current at 1449938 (00000000000000da53cb40da20776312c728970d5bb81ea7c942b1b679ea3e0f), subscribing to block notifications +0ms
  zap:lnd [INF]  2018-12-29 08:45:31.772 [INF] CRTR: Filtering chain using 4755 channels active +285ms
  zap:lnd [INF]  2018-12-29 08:45:31.776 [INF] CRTR: Prune tip for Channel Graph: height=1449938, hash=00000000000000da53cb40da20776312c728970d5bb81ea7c942b1b679ea3e0f +3ms
  zap:lnd [INF]  2018-12-29 08:45:31.809 [INF] CMGR: Server listening on [::]:9735 +33ms
  zap:lnd [INF]  2018-12-29 08:45:31.817 [INF] BTCN: Lost peer 35.231.86.179:18333 (outbound) +9ms
  zap:lnd [INF]  2018-12-29 08:45:31.819 [INF] BTCN: Syncing to block height 1449939 from peer 35.196.85.107:18333 +1ms
  zap:lnd [INF]  2018-12-29 08:45:31.819 [INF] BTCN: Fetching set of headers from tip (height=1449939) from peer 35.196.85.107:18333 +1ms
  zap:lnd [INF]  2018-12-29 08:45:31.819 [INF] BTCN: Lost peer 35.196.85.107:18333 (outbound) +0ms
  zap:lnd [WRN]  2018-12-29 08:45:31.819 [WRN] BTCN: No sync peer candidates available +0ms
  zap:lnd [INF]  2018-12-29 08:45:31.869 [INF] BTCN: Block manager shutting down +49ms
  zap:lnd [INF]  2018-12-29 08:45:31.869 [INF] BTCN: Address manager shutting down +1ms
  zap:lnd [INF]  2018-12-29 08:45:31.880 [INF] LTND: Shutdown complete +11ms
  zap:lnd [ERR]  panic: runtime error: invalid memory address or nil pointer dereference +2ms
  zap:lnd [ERR]  [signal SIGSEGV: segmentation violation code=0x1 addr=0x60 pc=0x193663a] +1ms
  zap:lnd [ERR]   +0ms
  zap:lnd [ERR]  goroutine 1 [running]: +0ms
  zap:lnd [ERR]  main.(*server).establishPersistentConnections.func1(0xc000320d20, 0xc000486420, 0x0, 0xc000276480, 0xc000486478, 0x21) +0ms
  zap:lnd [ERR]  	/Users/tom/.go/src/github.com/lightningnetwork/lnd/server.go:1649 +0x3a +1ms
  zap:lnd [ERR]  github.com/lightningnetwork/lnd/channeldb.(*LightningNode).ForEachChannel.func1(0xc000320d20, 0x1d39c88, 0xc000320d20) +0ms
  zap:lnd [ERR]  	/Users/tom/.go/src/github.com/lightningnetwork/lnd/channeldb/graph.go:2017 +0x591 +0ms
  zap:lnd [ERR]  github.com/lightningnetwork/lnd/vendor/github.com/coreos/bbolt.(*DB).View(0xc0000b41e0, 0xc000261a70, 0x0, 0x0) +1ms
  zap:lnd [ERR]  	/Users/tom/.go/src/github.com/lightningnetwork/lnd/vendor/github.com/coreos/bbolt/db.go:699 +0x90 +0ms
  zap:lnd [ERR]  github.com/lightningnetwork/lnd/channeldb.(*LightningNode).ForEachChannel(0xc00051a3c0, 0x0, 0xc00025d020, 0x0, 0xc010946978) +0ms
  zap:lnd [ERR]  	/Users/tom/.go/src/github.com/lightningnetwork/lnd/channeldb/graph.go:2029 +0xbf +0ms
  zap:lnd [ERR]  main.(*server).establishPersistentConnections(0xc0004da000, 0x0, 0x0) +0ms
  zap:lnd [ERR]  	/Users/tom/.go/src/github.com/lightningnetwork/lnd/server.go:1644 +0x33e +0ms
  zap:lnd [ERR]  main.(*server).Start(0xc0004da000, 0x1bf4411, 0x2e) +0ms
  zap:lnd [ERR]  	/Users/tom/.go/src/github.com/lightningnetwork/lnd/server.go:991 +0x231 +0ms
  zap:lnd [ERR]  main.lndMain(0x0, 0x0) +0ms
  zap:lnd [ERR]  	/Users/tom/.go/src/github.com/lightningnetwork/lnd/lnd.go:417 +0x10b8 +0ms
  zap:lnd [ERR]  main.main() +0ms
  zap:lnd [ERR]  	/Users/tom/.go/src/github.com/lightningnetwork/lnd/lnd.go:449 +0x26 +0ms
  zap:main [DBG]  Neutrino process received "exit" event with code 2 and signal null +44ms
  zap:main [INF]  Lnd process has shut down (code: 2, signal: null) +1ms
  zap:main [INF]  Sending message to renderer process: { msg: 'lndStopped', data: undefined } +0ms
  zap:main [DBG]  [FSM] onTerminated... +5s
  zap:main [DBG]  [FSM] onTerminate... +1ms
  zap:main [WRN]  Unable to fetch block height from https://tchain.api.btc.com/v3/block/latest: timeout of 5000ms exceeded +61ms
Segmentation fault: 11

**Comment by qjawe** [comment]
What when wrong?

**Comment by patmillertime** [comment]
> Thanks @mrfelton. I will do this when I get home this evening and report back!

Here's the results of my debug log.  Error appears to be: zap:lnd [ERR]  lookup localhost: no such host 

`(node:6688) ExperimentalWarning: The http2 module is an experimental API.
  zap:main [LOG]  Time until app is ready: 61.870358ms +0ms
  zap:main [DBG]  Fetching user settings from indexedDb (using database "ZapDesktop.production") +34ms
  zap:main [DBG]  Got user settings: [ { key: 'activeWallet', value: null }, { key: 'isWalletOpen', value: false }, { key: 'theme', value: 'dark' } ] +208ms
2018-12-28 16:58:59.651 ZapDesktop[6688:554500] *** WARNING: Textured window <AtomNSWindow: 0x7fd3d2a54fd0> is getting an implicitly transparent titlebar. This will break when linking against newer SDKs. Use NSWindow's -titlebarAppearsTransparent=YES instead.
  zap:updater [INF]  Checking for update +14ms
  zap:i18n [ 'Determined language as +0ms', 'Bulgarian' ]
  zap:i18n [ 'Determined language as +1ms', 'Czech' ]
  zap:i18n [ 'Determined language as +0ms', 'German' ]
  zap:i18n [ 'Determined language as +1ms', 'English' ]
  zap:i18n [ 'Determined language as +0ms', 'Castilian' ]
  zap:i18n [ 'Determined language as +0ms', 'French' ]
  zap:i18n [ 'Determined language as +1ms', 'Irish' ]
  zap:i18n [ 'Determined language as +6ms', 'Croatian' ]
  zap:i18n [ 'Determined language as +0ms', 'Japanese' ]
  zap:i18n [ 'Determined language as +1ms', 'Dutch' ]
  zap:i18n [ 'Determined language as +0ms', 'Portuguese' ]
  zap:i18n [ 'Determined language as +0ms', 'Moldavian' ]
  zap:i18n [ 'Determined language as +0ms', 'Russian' ]
  zap:i18n [ 'Determined language as +0ms', 'Swedish' ]
  zap:i18n [ 'Determined language as +0ms', 'Turkish' ]
  zap:i18n [ 'Determined language as +0ms', 'Ukrainian' ]
  zap:main [DBG]  [FSM] onOnboarding... +541ms
  zap:main [DBG]  [FSM] onStartOnboarding... +202ms
  zap:main [INF]  Sending message to renderer process: { msg: 'startOnboarding', data: undefined } +0ms
  zap:updater [INF]  Update for version 0.3.2-beta is not available (latest version: 0.3.2-beta, downgrade is disallowed). +119ms
  zap:main [INF]  Starting lnd with options: { chain: 'bitcoin', id: 'tmp', network: 'testnet', type: 'local' } +27s
  zap:lnd-config Constructor called with options: { id: 'tmp', type: 'local', chain: 'bitcoin', network: 'testnet', settings: {} } +0ms
  zap:lnd-config Setting settings as: { name: null, autopilot: true, autopilotMaxchannels: 5, autopilotMinchansize: 20000, autopilotMaxchansize: 16777215, autopilotAllocation: 0.6, autopilotPrivate: true, autopilotMinconfs: 0 } +0ms
  zap:lnd-config Connection type is local. Assigning settings as: { host: 'localhost:10009', cert: '/Users/Patrick/Library/Application Support/Zap/lnd/bitcoin/testnet/wallet-tmp/tls.cert', macaroon: '/Users/Patrick/Library/Application Support/Zap/lnd/bitcoin/testnet/wallet-tmp/data/chain/bitcoin/testnet/admin.macaroon' } +1ms
  zap:main [DBG]  [FSM] onBeforeStartLocalLnd... +2ms
  zap:main [INF]  Starting new lnd instance +0ms
  zap:main [INF]   > alias: undefined +0ms
  zap:main [INF]   > autopilot: true +0ms
  zap:main [INF]  Starting Neutrino... +1ms
  zap:main [INF]  Starting lnd in neutrino mode +0ms
  zap:main [INF]   > binaryPath /Applications/ZapDesktop.app/Contents/Resources/bin/lnd +0ms
  zap:main [INF]   > rpcProtoPath: /Applications/ZapDesktop.app/Contents/Resources/resources/rpc.proto +1ms
  zap:main [INF]   > host: localhost:10009 +0ms
  zap:main [INF]   > cert: /Users/Patrick/Library/Application Support/Zap/lnd/bitcoin/testnet/wallet-tmp/tls.cert +0ms
  zap:main [INF]   > macaroon: /Users/Patrick/Library/Application Support/Zap/lnd/bitcoin/testnet/wallet-tmp/data/chain/bitcoin/testnet/admin.macaroon +0ms
  zap:main [INF]  Spawning Neutrino process: /Applications/ZapDesktop.app/Contents/Resources/bin/lnd --configfile=/Applications/ZapDesktop.app/Contents/Resources/resources/lnd.conf --lnddir=/Users/Patrick/Library/Application Support/Zap/lnd/bitcoin/testnet/wallet-tmp --listen=0.0.0.0:9735 --rpclisten=localhost:11009 --restlisten=localhost:8180 --autopilot.active --autopilot.private --autopilot.maxchannels=5 --autopilot.minchansize=20000 --autopilot.maxchansize=16777215 --autopilot.allocation=0.6 --autopilot.minconfs=0 --neutrino.connect=testnet1-btcd.zaphq.io --neutrino.connect=testnet2-btcd.zaphq.io +4ms
  zap:main [INF]  Sending message to renderer process: { msg: 'lndStarted', data: e { id: 'tmp', type: 'local', chain: 'bitcoin', network: 'testnet', host: [Getter/Setter], cert: [Getter/Setter], macaroon: [Getter/Setter], string: [Getter/Setter], name: null, alias: undefined, autopilot: true, autopilotMaxchannels: 5, autopilotMinchansize: 20000, autopilotMaxchansize: 16777215, autopilotAllocation: 0.6, autopilotPrivate: true, autopilotMinconfs: 0, wallet: [Getter], binaryPath: [Getter], lndDir: [Getter], configPath: [Getter], rpcProtoPath: [Getter] } } +8ms
  zap:lnd [ERR]  lookup localhost: no such host +42ms
  zap:main [DBG]  Neutrino process received "exit" event with code 1 and signal null +3ms
  zap:main [INF]  Lnd process has shut down (code: 1, signal: null) +0ms
  zap:main [INF]  Sending message to renderer process: { msg: 'lndStopped', data: undefined } +1ms
  zap:main [DBG]  [FSM] onTerminated... +4m
  zap:main [DBG]  [FSM] onTerminate... +1ms`

**Comment by qjawe** [comment]
> zap:lnd [ERR] lookup localhost: no such host

That very strange, how could I restart the LND host?

**Comment by qjawe** [comment]
> zap:lnd [INF] 2018-12-29 08:45:31.472 [INF] CRTR: FilteredChainView starting +0ms
> zap:lnd [ERR] 2018-12-29 08:45:31.481 [ERR] DISC: Unable to rebroadcast stale channels: unable to retrieve outgoing channels: Channel from self node has no policy +1ms
> zap:lnd [DBG] 2018-12-29 08:45:31.487 [DBG] BTCN: Waiting for filter headers (height=1449938) to catch up the rescan start (height=1449938) +0ms
> zap:lnd [DBG] 2018-12-29 08:45:31.487 [DBG] BTCN: Starting rescan from known block 1449938 (00000000000000da53cb40da20776312c728970d5bb81ea7c942b1b679ea3e0f) +0ms
> zap:lnd [DBG] 2018-12-29 08:45:31.487 [DBG] BTCN: Rescan became current at 1449938 (00000000000000da53cb40da20776312c728970d5bb81ea7c942b1b679ea3e0f), subscribing to block notifications +0ms
> zap:lnd [INF] 2018-12-29 08:45:31.772 [INF] CRTR: Filtering chain using 4755 channels active +285ms
> zap:lnd [INF] 2018-12-29 08:45:31.776 [INF] CRTR: Prune tip for Channel Graph: height=1449938, hash=00000000000000da53cb40da20776312c728970d5bb81ea7c942b1b679ea3e0f +3ms
> zap:lnd [INF] 2018-12-29 08:45:31.809 [INF] CMGR: Server listening on [::]:9735 +33ms
> zap:lnd [INF] 2018-12-29 08:45:31.817 [INF] BTCN: Lost peer 35.231.86.179:18333 (outbound) +9ms
> zap:lnd [INF] 2018-12-29 08:45:31.819 [INF] BTCN: Syncing to block height 1449939 from peer 35.196.85.107:18333 +1ms
> zap:lnd [INF] 2018-12-29 08:45:31.819 [INF] BTCN: Fetching set of headers from tip (height=1449939) from peer 35.196.85.107:18333 +1ms
> zap:lnd [INF] 2018-12-29 08:45:31.819 [INF] BTCN: Lost peer 35.196.85.107:18333 (outbound) +0ms
> zap:lnd [WRN] 2018-12-29 08:45:31.819 [WRN] BTCN: No sync peer candidates available +0ms
> zap:lnd [INF] 2018-12-29 08:45:31.869 [INF] BTCN: Block manager shutting down +49ms
> zap:lnd [INF] 2018-12-29 08:45:31.869 [INF] BTCN: Address manager shutting down +1ms
> zap:lnd [INF] 2018-12-29 08:45:31.880 [INF] LTND: Shutdown complete +11ms
> zap:lnd [ERR] panic: runtime error: invalid memory address or nil pointer dereference +2ms
> zap:lnd [ERR] [signal SIGSEGV: segmentation violation code=0x1 addr=0x60 pc=0x193663a] +1ms
> zap:lnd [ERR] +0ms
> zap:lnd [ERR] goroutine 1 [running]: +0ms
> zap:lnd [ERR] main.(*server).establishPersistentConnections.func1(0xc000320d20, 0xc000486420, 0x0, 0xc000276480, 0xc000486478, 0x21) +0ms
> zap:lnd [ERR] /Users/tom/.go/src/github.com/lightningnetwork/lnd/server.go:1649 +0x3a +1ms
> zap:lnd [ERR] github.com/lightningnetwork/lnd/channeldb.(*LightningNode).ForEachChannel.func1(0xc000320d20, 0x1d39c88, 0xc000320d20) +0ms
> zap:lnd [ERR] /Users/tom/.go/src/github.com/lightningnetwork/lnd/channeldb/graph.go:2017 +0x591 +0ms
> zap:lnd [ERR] github.com/lightningnetwork/lnd/vendor/github.com/coreos/bbolt.(*DB).View(0xc0000b41e0, 0xc000261a70, 0x0, 0x0) +1ms
> zap:lnd [ERR] /Users/tom/.go/src/github.com/lightningnetwork/lnd/vendor/github.com/coreos/bbolt/db.go:699 +0x90 +0ms
> zap:lnd [ERR] github.com/lightningnetwork/lnd/channeldb.(*LightningNode).ForEachChannel(0xc00051a3c0, 0x0, 0xc00025d020, 0x0, 0xc010946978) +0ms
> zap:lnd [ERR] /Users/tom/.go/src/github.com/lightningnetwork/lnd/channeldb/graph.go:2029 +0xbf +0ms
> zap:lnd [ERR] main.(*server).establishPersistentConnections(0xc0004da000, 0x0, 0x0) +0ms
> zap:lnd [ERR] /Users/tom/.go/src/github.com/lightningnetwork/lnd/server.go:1644 +0x33e +0ms
> zap:lnd [ERR] main.(*server).Start(0xc0004da000, 0x1bf4411, 0x2e) +0ms
> zap:lnd [ERR] /Users/tom/.go/src/github.com/lightningnetwork/lnd/server.go:991 +0x231 +0ms
> zap:lnd [ERR] main.lndMain(0x0, 0x0) +0ms
> zap:lnd [ERR] /Users/tom/.go/src/github.com/lightningnetwork/lnd/lnd.go:417 +0x10b8 +0ms
> zap:lnd [ERR] main.main() +0ms
> zap:lnd [ERR] /Users/tom/.go/src/github.com/lightningnetwork/lnd/lnd.go:449 +0x26 +0ms
> zap:main [DBG] Neutrino process received "exit" event with code 2 and signal null +44ms
> zap:main [INF] Lnd process has shut down (code: 2, signal: null) +1ms
> zap:main [INF] Sending message to renderer process: { msg: 'lndStopped', data: undefined } +0ms
> zap:main [DBG] [FSM] onTerminated... +5s
> zap:main [DBG] [FSM] onTerminate... +1ms
> zap:main [WRN] Unable to fetch block height from https://tchain.api.btc.com/v3/block/latest: timeout of 5000ms exceeded +61ms
> Segmentation fault: 11

<img width="988" alt="screenshot 2018-12-29 09 11 30" src="https://user-images.githubusercontent.com/16167774/50531713-d2489100-0b49-11e9-8e12-5a40ffe2a47d.png">
Delete Wallet and resync Wallet again let's see how it goes?

**Comment by qjawe** [comment]
After deleting

 wallet and sync back Testnet funds are zero

**Comment by qjawe** [comment]
> After deleting
> 
> wallet and sync back Testnet funds are zero

I was suprised the old BTC address was gone!! When i restore the wallet!!

**Comment by mrfelton** [comment]
> > After deleting
> > wallet and sync back Testnet funds are zero
> 
> I was suprised the old BTC address was gone!! When i restore the wallet!!

If you restore a wallet from its seed, you will get a fresh BTC address - this is expected. Additionally, you will get a fresh BTC address every time you send finds to an address as addresses are supposed to be single use only.

**Comment by mrfelton** [comment]
> Here's the results of my debug log. Error appears to be: zap:lnd [ERR] lookup localhost: no such host

@patmillertime it sounds like you don't have your hosts file set up correctly.

Make sure this line is included in the hosts file
```
127.0.0.1 localhost
```

For MacOS you can find the hosts file here: /etc/hosts

See http://osxdaily.com/2014/04/12/restore-original-hosts-file-mac/

**Comment by qjawe** [comment]
> BTC address every time you send funds to an ad

Recreated a new wallet address: https://live.blockcypher.com/btc-testnet/address/2MyHh4UgayWPyP2bzJZ19XK5iFoCxmbSAeM/

**Comment by qjawe** [comment]
> > > After deleting
> > > wallet and sync back Testnet funds are zero
> > 
> > 
> > I was surprised the old BTC address was gone!! When i restore the wallet!!
> 
> @qjawe If you restore a wallet from its seed, you will get a fresh BTC address - this is expected. Additionally, you will get a fresh BTC address every time you send funds to an address as addresses are supposed to be single use only.

If that a live mainnet which the old funds on the old BTC address will be lost also due to the new address was been created, which does not make any sense

**Comment by mrfelton** [comment]
@qjawe the creation of a new address does not have any impact on old used addresses from the same seed. Funds associated with old addresses should be imported into the wallet.

Were your funds in the on-chain wallet address? Or were they all locked into channels?

**Comment by qjawe** [comment]
> @qjawe the creation of a new address does not have any impact on old used addresses from the same seed. Funds associated with old addresses should be imported into the wallet.
> 
> Were your funds in the on-chain wallet address? Or were they all locked into channels?

@mrfelton fund did import to the new address, this is what happens
Open up Bitfury Peach wallet at the same time and ZAP kept crashing on LND. Rebooted the company also and found the new imported wallet had zero funds and all the old channel had been lost and funds also been lost on the new wallet

New create a new wallet and see how it goes.

Reinstall ZAP desktop on MAC also

**Comment by mrfelton** [comment]
@qjawe importing a wallet from a seed does not restore channels. Only on-chain funds can be restored with the seed unfortunately. This is a current limitation in lnd.

**Comment by qjawe** [comment]
> store channels. Only on-chain funds ca

understood and if the funded on the LND BTC will be lost once the owner importing a wallet from a seed also?

**Comment by patmillertime** [comment]
> > Here's the results of my debug log. Error appears to be: zap:lnd [ERR] lookup localhost: no such host
> 
> @patmillertime it sounds like you don't have your hosts file set up correctly.
> 
> Make sure this line is included in the hosts file
> 
> ```
> 127.0.0.1 localhost
> ```
> For MacOS you can find the hosts file here: /etc/hosts
> 
> See http://osxdaily.com/2014/04/12/restore-original-hosts-file-mac/

Thank you @mrfelton.

The hosts file was completely blank for some reason. Updating the hosts file with the localhost IP address solved the problem.
